packages/block-library/src/social-links/edit.js

```diff
@@ -80,7 +81,6 @@ export function SocialLinksEdit( props ) {
 
 	const SocialPlaceholder = (
 		<li className="wp-block-social-links__social-placeholder">
-			<div className="wp-social-link"></div>
```

Comment:
**Question:** Was this leftover from previous commits?

Reply:
Yeah I believe that was leftover. It does not render anything, so I figured I would just fix it. I can move to a separate PR if its inclusion here is confusing.

Reply:
It’s okay to remove it here 👍